src/lib/annotations/doc/CreateHighlightDialog.js

```diff
@@ -323,6 +344,7 @@ class CreateHighlightDialog extends EventEmitter {
         highlightDialogEl.addEventListener('click', this.stopPropagation);
         highlightDialogEl.addEventListener('mouseup', this.stopPropagation);
         highlightDialogEl.addEventListener('touchend', this.stopPropagation);
+        highlightDialogEl.addEventListener('touchstart', this.stopPropagation);
```

Comment:
What's happening on touchstart that isn't stopped by all the other stop propagation we've done?